sys/include/net/gnrc/lwmac/lwmac.h

```diff
@@ -31,35 +31,61 @@ extern "C" {
 #endif
 
 /**
- * @brief Time between consecutive wakeups. This parameter governs power consumption,
- *        latency and throughput!
+ * @brief Time between consecutive wake-ups.
```

Comment:
nitpicky, but please add a newline between `@brief` line and detailed description (and in my opinion the detailed description doesn't need indentation)